tests/tests.js

```diff
@@ -463,10 +463,15 @@ QUnit.module( "CheckIt tests dropdown", {
 
 
 QUnit.test("addCalendarToDropdown test", function( assert ) {
-    assert.expect(2);
+    assert.expect(3);
     this.checkit.addCalendarToDropdown("1234", "Hello", this.$dropdown);
     assert.equal(this.$dropdown.children().attr('id'), '1234');
     assert.equal(this.$dropdown.children().text(), "Hello");
+    
+    // Check with items in dropdown already.
```

Comment:
I meant the previous asserts with something in the dropdown already. I suggest.

1. Add made up thing to dropdown
2. Check that 'hello' whatever isn't there.
3. Add "1234", "Hello" to dropdown.
4. Test that it *is* there.